solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -69,30 +70,30 @@ def create_dictionary(path_of_file):
 
     Args:
         path_of_file (string): Path of the file that contains all bank
-        costumer data.
+            costumer data.
 
     Returns:
-        dictionary: A nested dictionary of bank costumer data, or an exception
-        if the file path is not correct.
+        dictionary: A nested dictionary of bank costumer data, or print a
+            message and exit if the file path is not correct.
     """
-    try:
+
+    if os.path.isfile(path_of_file):
         with open(path_of_file, 'r') as file_j:
             data = json.load(file_j)
             costumers_data = data
             return costumers_data
 
-    except FileNotFoundError:
-        return "Your file path is incorrect. Please check it."
+    print("Your file path is incorrect. Please check it.")
+    quit()
```

Comment:
You almost never want to run "quit" in a function.
Think about that you are working with calculator, and if you tried to divide by 0 the calculator will close